src/Traisi/Controllers/SurveyAnalyticsController.cs

```diff
@@ -158,130 +150,181 @@ await this
             {
                 //QuestionType Responses
                 var questTypeResponses =
-                    (await surveyResponseValues
+                    (
+                    await surveyResponseValues
                         .Where(item =>
                             item.SurveyResponse.QuestionPart.Id == questionId &&
                             item.SurveyResponse.QuestionPart.SurveyId ==
-                            surveyId).ToListAsync()).Select(p => new
+                            surveyId)
+                        .ToListAsync()
+                    )
+                        .Select(p =>
+                        {
+                            if (((OptionSelectResponse) p).Code != "nota")
```

Comment:
still a hard coded "nota"

Reply:
Fixed the hard coded "nota" and pushed changes